.env.development

```diff
@@ -1,2 +1,2 @@
 REACT_EDITOR=code-insiders
-REACT_APP_ENV=development
```

Comment:
That would be great not to be committed :)

Reply:
Branch has been updated without the `package-lock.json` and `.env.development` updates